lib/broadway_kafka.ex

```diff
@@ -24,7 +24,7 @@ defmodule BroadwayKafka do
 
   defp each_producer(server, fun) when is_function(fun, 1) do
     server
-    |> Broadway.Server.producer_names()
+    |> Broadway.Topology.producer_names()
```

Comment:
We should not be using private APIs. Would simple Broadway work instead?

Reply:
You're totally correct, didn't knew this function exist. Going to push an update.